insights/specs/__init__.py

```diff
@@ -307,6 +307,10 @@ class Specs(SpecSet):
     os_release = RegistryPoint()
     ovirt_engine_confd = RegistryPoint(multi_output=True)
     ovirt_engine_server_log = RegistryPoint()
+    # ovirt_engine_log = RegistryPoint()
+    ovirt_engine_ui_log = RegistryPoint()
```

Comment:
@psachin I don't think we want to have commented out specs here because it would not be clear why they are commented out.  I'm also working with @csams to clarify whether or not we can have two specs collect the same file, and if so how we should mark the old spec as deprecated.

If you want this PR to go ahead and be merged now, we can deal with the spec deprecation in a separate PR, you just need to remove the commented specs.  Otherwise this PR will need to wait until we can determine how to deprecate specs.

Reply:
Sure